docs/cli/argocd_account.md

```diff
@@ -0,0 +1,91 @@
+# `argocd account` Command Reference
+
+## argocd account
+
+Manage account settings
+
+```
+argocd account [flags]
+```
+
+### Examples
+
+```
+  # List accounts
+  argocd account list
+  
+  # Update the current user's password
+  argocd account update-password
+  
+  # Can I sync any app?
+  argocd account can-i sync applications '*'
+  
+  # Get User information
+  argocd account get-user-info
+```
+
+### Options
```

Comment:
@anandf Suggest using this model for headers for all the relevant tables in this CLI reference docs (Global comment).

Reply:
Added tables for all global and sub command specific options.